src/ocm/components/clusterConfiguration/staticIp/components/FormViewNetworkWide/formViewNetworkWideValidationSchema.tsx

```diff
@@ -12,6 +12,7 @@ import {
   getIpIsNotNetworkOrBroadcastAddressSchema,
 } from '../../commonValidationSchemas';
 const REQUIRED_MESSAGE = 'A value is required';
+const REQUIRED_MESSAGE_AND_MUST_BE_A_NUMBER = 'A value is required and must be a number';
```

Comment:
Works well, but I think the message could be better. 
I suggest

"A numeric value is required" or
"A value is required, and it must be a number"

but we can double-check with @nirfarkas

Reply:
According to PF it should be:
"Must be a number"
![image](https://user-images.githubusercontent.com/65043359/190343766-d568e676-f880-4022-b2fa-c454ad1f7571.png)